frontend/src/views/AboutView.vue

```diff
@@ -1,3 +1,70 @@
 <template>
-  <div>About</div>
+  <v-container>
+    <div id="about">
+      <h1>{{ titlePage }}</h1>
+      <h2>{{ projectName }}</h2>
+      <br />
+      <p>
+        ft_transcendence is the last project of the 42 common core. It is
+        designed to complete the first part of our cursus. We had to program, as
+        a team, something completely different from what we have done so far. We
+        were supposed to create a website where users can face each other in
+        Pong games. The website where you are now! <br />The specifications
+        included: a nice user interface, a chat, and real-time multiplayer
+        online games, compatible with the latest stable up-to-date version of
+        Google Chrome, Firefox, and Safari. <br />We have developed our website
+        with these technologies:
+      </p>
+      <ul>
+        <li>backend: NestJS;</li>
+        <li>frontend: TypeScript, with Vue.js as framework;</li>
+        <li>database: PostgreSQL.</li>
+      </ul>
+      <br />
+      <p>
+        For more information about the rules we had to follow, you can check our
+        <a
+          href="https://github.com/fyusuf-a/ft_transcendence/blob/master/README.md"
+          target="_blank"
+          >README.md</a
+        >.
+      </p>
+    </div>
+    <div>
+      <h2>{{ team }}</h2>
+      <br />
+      <v-app id="inspire">
+        <v-container class="grey lighten-5">
+          <team-card :members="member" />
+        </v-container>
+      </v-app>
+    </div>
+  </v-container>
 </template>
+
+<script>
+import TeamCard from "@/components/Team/TeamCard.vue";
+
+export default {
+  components: {
+    "team-card": TeamCard,
+  },
+  data: function () {
+    return {
+      titlePage: "About",
+      projectName: "ft_transcendence",
+      team: "Our team",
+    };
+  },
+  methods: {},
```

Comment:
Except if we need to reuse this view (which I think we will not), it is not mandatory to have this information in variables. In terms of speed, even though here it is probably negligible, loading pure html is quicker than loading through javascript.

Reply:
Ok, I change it.
I didn't know that there were better performances with pure html. Duly noted, thank you!